apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.csproj

```
@@ -13,6 +13,6 @@
     <PackageReference Include="Google.Cloud.DevTools.Common" Version="[3.0.0, 4.0.0)" />
     <PackageReference Include="Google.Cloud.Logging.V2" Version="[4.0.0, 5.0.0)" />
     <PackageReference Include="Grpc.Core" Version="[2.46.6, 3.0.0)" PrivateAssets="None" Condition="'$(TargetFramework)'=='net462'" />
-    <PackageReference Include="NLog" Version="4.7.15" />
+    <PackageReference Include="NLog" Version="5.2.8" />
```

Maybe? Although I'd understand very well if we didn't.

```suggestion
    <PackageReference Include="NLog" Version="[5.2.8, 6.0.0)" />
```

---

We *typically* don't do that for third-party dependencies, as we don't know how firmly SemVer-oriented they are.
Currently our tooling won't handling putting a range in the API catalog either.